integrations/object_store/Cargo.toml

```diff
@@ -19,20 +19,20 @@
 description = "OpenDAL's object_store Integration"
 name = "object_store_opendal"
 
-authors.workspace = true
-edition.workspace = true
-homepage.workspace = true
-license.workspace = true
-repository.workspace = true
-rust-version.workspace = true
-version.workspace = true
+authors = ["Apache OpenDAL <dev@opendal.apache.org>"]
+edition = "2021"
+homepage = "https://opendal.apache.org/"
+license = "Apache-2.0"
+repository = "https://github.com/apache/opendal"
+rust-version = "1.67"
+version = "0.42.0"
 
 [dependencies]
 async-trait = "0.1"
 bytes = "1"
 futures = "0.3"
 object_store = "0.7"
-opendal.workspace = true
+opendal = {path = "../../core"}
```

Comment:
Ditto https://github.com/apache/opendal/pull/4134/files#r1477020121.

Reply:
The same with https://github.com/apache/opendal/pull/4134#discussion_r1477048529